actions/run_fuzzers/action.yml

```diff
@@ -74,7 +74,7 @@ inputs:
     default: true
 runs:
   using: 'docker'
-  image: 'docker://gcr.io/oss-fuzz-base/clusterfuzzlite-run-fuzzers:v1'
+  image: 'docker://gcr.io/oss-fuzz-base/clusterfuzzlite-run-fuzzers:ubuntu-24-04-v1'
```

Comment:
Is this change safe to make? Or do we pivot back to 20 if they aren't opting into 24?